backend/tests/regression/answer_quality/launch_search_test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+export PYTHONPATH="$(cd "$(dirname "${BASH_SOURCE[0]}")/../../../.." && pwd):$PYTHONPATH"
+export MODEL_SERVER_HOST="18.218.165.17" 
+export MODEL_SERVER_PORT="80" 
+
+python3.11 backend/tests/regression/answer_quality/search_quality_test.py
```

Comment:
I'm sure you would have cleaned it up prior to the final PR but let's always be super detail oriented, newline at end of file!